app/sdil/connectors/SoftDrinksIndustryLevyConnector.scala

```diff
@@ -51,7 +51,7 @@ class SoftDrinksIndustryLevyConnector(
 
   def checkPendingQueue(utr: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     http.GET[HttpResponse](s"$sdilUrl/check-enrolment-status/$utr") recover {
-      case _: NotFoundException => HttpResponse(404)
+      case _: NotFoundException => HttpResponse(404, "")
```

Comment:
We shouldn't use anymore Not Found Exception because it will fail. Use upstream error response